cmd/kube-apiserver/app/options/validation.go

```diff
@@ -150,6 +150,19 @@ func validateAPIPriorityAndFairness(options *ServerRunOptions) []error {
 	return nil
 }
 
+func validateAPIServerIdentity(options *ServerRunOptions) []error {
+	var errs []error
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServerIdentity) {
```

Comment:
It's unnecessary to check the feature gate.

Reply:
two flags valid when the feature gate enabled, so i add the logic.

Reply:
fixed